build.sbt

```diff
@@ -35,10 +38,12 @@ lazy val root = (project in file("."))
     ScoverageKeys.coverageHighlighting := true,
     scalacOptions ++= Seq("-feature"),
     libraryDependencies ++= AppDependencies(),
+    libraryDependencies ++= Seq(
+      compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.6.0" cross CrossVersion.full),
```

Comment:
please don't have duplicate keys here and add all the dependencies inside AppDependencies. Also for what exactly is used this silencer?

Reply:
These to be moved in AppDependencies